packages/contentful-migrations/ui/1000000000004-create-c-editorial-migration.js

```diff
@@ -3,28 +3,31 @@ module.exports = async function (migration) {
     .createContentType('c-editorial')
     .name('Component: Editorial')
     .description('Teaser-like components with text, image & links')
-    .displayField('name');
+    .displayField('internal_name');
+
   cEditorial
-    .createField('name')
+    .createField('internal_name')
     .name('Internal name')
     .type('Symbol')
     .localized(false)
     .required(true)
     .validations([])
     .disabled(false)
     .omitted(false);
+
   cEditorial
-    .createField('title')
-    .name('Title')
+    .createField('headline')
+    .name('Headline')
     .type('Symbol')
     .localized(true)
     .required(false)
     .validations([])
     .disabled(false)
     .omitted(false);
+
   cEditorial
-    .createField('subtitle')
-    .name('Subtitle')
+    .createField('secondaryHeadline')
```

Comment:
a) Why not `subheadline`?
b) snake_case ;)

Reply:
b) done
a) Actually, you are right. But there is often a change in the position of this headline in the design. Sometimes it is before and sometimes after the primary headline. Is "overline" then correct? I wanted to avoid this discussion with the "secondary headline". I'll ask our designers again.

Reply:
We have now a `subline` and a `overline`. Both are used by default as `div` tags. Depending on the use case, it can then be decided which of the two is the semantically more important headline.